src/sphinx_antsibull_ext/css/antsibull-minimal.scss

```diff
@@ -1,4 +1,6 @@
 /* Copyright (c) Ansible and contributors */
+/* GNU General Public License v3.0+ (see https://www.gnu.org/licenses/gpl-3.0.txt) */
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Different comment styles

Reply:
`/* */` comments are included in the generated CSS files, `//` comments are not. This is important here since `SPDX-License-Identifier` would end up in a way that REUSE cannot use (because there's something after the license name in the same line), but we do want the copyright and readable license information in the generated file.

Reply:
Resolve